webcompat/webhooks/helpers.py

```diff
@@ -78,3 +82,25 @@ def get_payload_signature(key, payload):
     '''Compute the payload signature given a key.'''
     mac = hmac.new(key, msg=payload, digestmod=hashlib.sha1)
     return mac.hexdigest()
+
+
+def extract_priority_label(body):
```

Comment:
No blank lines in function code now. Wait topsites.py to deploy and make a new DB, then we can merge this PR.